sdk/include/opentelemetry/sdk/metrics/meter.h

```diff
@@ -105,7 +106,11 @@ class Meter final : public opentelemetry::metrics::Meter
   // order of declaration is important here - instrumentation library should destroy after
   // meter-context.
   std::unique_ptr<sdk::instrumentationlibrary::InstrumentationLibrary> instrumentation_library_;
-  std::shared_ptr<sdk::metrics::MeterContext> context_;
+  std::shared_ptr<sdk::metrics::MeterContext> meter_context_;
+  std::unordered_map<std::string, std::shared_ptr<MetricStorage>> storage_registry_;
+
+  std::unique_ptr<WritableMetricStorage> RegisterMetricStorage(
```

Comment:
nit: I think for private methods we should use `registerMetricStorage`?

Reply:
Google-style [guide](https://google.github.io/styleguide/cppguide.html#Function_Names) doesn't seem to differentiate in naming private and public methods. I have been sticking to `CamelCase` for both private and public. Looking into the repo, I think we are not consistent and have been using both `CamelCase` and `camelCase`. Probably good to discuss and use one form.